### ISSUE TITLE
./configure fails on Ubuntu 17.10

### ISSUE THREAD

**Issue by stuaxo** [redis/RedisDesktopManager#4015]
Pretty sure I've built this successfully before, but now I'm getting an error from ```ar``` when I run ```./configure```

```ar: `u' modifier ignored since `D' is the default (see `U')```

This is from following the instructions on the website about building

```
$ ./configure
...[snipped]....
config.status: creating breakpad-client.pc
config.status: creating Makefile
config.status: creating src/config.h
config.status: executing depfiles commands
src/third_party/libdisasm/x86_format.c: In function ‘format_operand_xml.isra.3’:
src/third_party/libdisasm/x86_format.c:837:5: warning: ‘%04X’ directive output truncated writing 4 bytes into a region of size 1 [-Wformat-truncation=]
     "\t\t<absolute_address segment=\"0x%04" PRIX16 "\"",
     ^
src/third_party/libdisasm/x86_format.c:34:38: note: in definition of macro ‘STRNCATF’
         snprintf( _tmp, sizeof _tmp, fmt, data );   \
                                      ^~~
src/third_party/libdisasm/x86_format.c:837:40: note: format string is defined here
     "\t\t<absolute_address segment=\"0x%04" PRIX16 "\"",
src/third_party/libdisasm/x86_format.c:837:5: note: directive argument in the range [0, 65535]
     "\t\t<absolute_address segment=\"0x%04" PRIX16 "\"",
     ^
src/third_party/libdisasm/x86_format.c:34:38: note: in definition of macro ‘STRNCATF’
         snprintf( _tmp, sizeof _tmp, fmt, data );   \
                                      ^~~
In file included from /usr/include/stdio.h:862:0,
                 from src/third_party/libdisasm/x86_format.c:1:
/usr/include/x86_64-linux-gnu/bits/stdio2.h:64:10: note: ‘__builtin___snprintf_chk’ output 37 bytes into a destination of size 32
   return __builtin___snprintf_chk (__s, __n, __USE_FORTIFY_LEVEL - 1,
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
        __bos (__s), __fmt, __va_arg_pack ());
        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/third_party/libdisasm/x86_format.c: In function ‘x86_format_insn’:
src/third_party/libdisasm/x86_format.c:1141:24: warning: ‘|’ directive output may be truncated writing 1 byte into a region of size between 0 and 31 [-Wformat-truncation=]
         STRNCATF( buf, "|%s|", insn->prefix_string             , len );
                        ^
src/third_party/libdisasm/x86_format.c:34:38: note: in definition of macro ‘STRNCATF’
         snprintf( _tmp, sizeof _tmp, fmt, data );   \
                                      ^~~
In file included from /usr/include/stdio.h:862:0,
                 from src/third_party/libdisasm/x86_format.c:1:
/usr/include/x86_64-linux-gnu/bits/stdio2.h:64:10: note: ‘__builtin___snprintf_chk’ output between 3 and 34 bytes into a destination of size 32
   return __builtin___snprintf_chk (__s, __n, __USE_FORTIFY_LEVEL - 1,
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
        __bos (__s), __fmt, __va_arg_pack ());
        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/third_party/libdisasm/x86_format.c:1181:24: warning: ‘%s’ directive output may be truncated writing up to 31 bytes into a region of size 22 [-Wformat-truncation=]
         STRNCATF( buf, "\" string=\"%s\"/>\n", insn->prefix_string, len );
                        ^
src/third_party/libdisasm/x86_format.c:34:38: note: in definition of macro ‘STRNCATF’
         snprintf( _tmp, sizeof _tmp, fmt, data );   \
                                      ^~~
In file included from /usr/include/stdio.h:862:0,
                 from src/third_party/libdisasm/x86_format.c:1:
/usr/include/x86_64-linux-gnu/bits/stdio2.h:64:10: note: ‘__builtin___snprintf_chk’ output between 15 and 46 bytes into a destination of size 32
   return __builtin___snprintf_chk (__s, __n, __USE_FORTIFY_LEVEL - 1,
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
        __bos (__s), __fmt, __va_arg_pack ());
        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
ar: `u' modifier ignored since `D' is the default (see `U')
ar: `u' modifier ignored since `D' is the default (see `U')
ar: `u' modifier ignored since `D' is the default (see `U')

```

**Comment by uglide** [comment]
@stuaxo try to reclone submodules if it's not clean build

**Comment by stuaxo** [comment]
I tried deleting the directory and redoing ```git clone --recursive https://github.com/uglide/RedisDesktopManager.git -b 0.9 rdm && cd ./rdm``` and it still fails

**Comment by stuaxo** [comment]
I think   ```ar: `u' modifier ignored since `D' is the default (see `U')```   might be something to do with deterministic builds, I'm finding the error related to all sorts of projects.

http://gnu-automake.7480.n7.nabble.com/bug-20082-new-warning-from-ar-on-rawhide-systems-td21769.html

**Comment by uglide** [comment]
Closed (since 18.04 is released and RDM builds on it just fine).
